src/hashTable.js

```diff
@@ -1,63 +1,94 @@
 class Node{
-  constructor() {
-    this.key = 0,
-    this.data = 0,
+  constructor(key, data) {
+    this.key = key,
+    this.data = data,
     this.next = null
   }
 }
 
 export default class Hashtable{
   constructor(){
-    this._length,
+    this._length = 0,
     this.dataStore = []
   }
 
-  hashFunction(key) {
+  //takes a key and returns a hash number which will be where the hash will live.
+  hashFunction(key){
     let sum = 0
     for (let i = 0; i < key.length; i++) {
       sum += key.charCodeAt(i)
     }
     return sum % 31
```

Comment:
haha I don't know it was fairly random and the reason I have so many puts is because I was testing the chaining and making sure it added accordingly in the right place.